sympy/physics/control/tests/test_lti.py

```diff
@@ -577,53 +574,186 @@ def test_Feedback_construction():
 
 
 def test_Feedback_functions():
-    zeta, wn = symbols('zeta, wn')
     tf = TransferFunction(1, 1, s)
-    tf1 = TransferFunction(1, s**2 + 2*zeta*wn*s + wn**2, s)
-    tf2 = TransferFunction(k, 1, s)
-    tf3 = TransferFunction(a2*p - s, a2*s + p, s)
     tf4 = TransferFunction(a0*p + p**a1 - s, p, p)
     tf5 = TransferFunction(a1*s**2 + a2*s - a0, s + a0, s)
     tf6 = TransferFunction(s - p, p + s, p)
 
-    assert tf / (tf + tf1) == Feedback(tf, tf1)
-    assert tf / (tf + tf1*tf2*tf3) == Feedback(tf, tf1*tf2*tf3)
-    assert tf1 / (tf + tf1*tf2*tf3) == Feedback(tf1, tf2*tf3)
-    assert (tf1*tf2) / (tf + tf1*tf2) == Feedback(tf1*tf2, tf)
-    assert (tf1*tf2) / (tf + tf1*tf2*tf5) == Feedback(tf1*tf2, tf5)
-    assert (tf1*tf2) / (tf + tf1*tf2*tf5*tf3) in (Feedback(tf1*tf2, tf5*tf3), Feedback(tf1*tf2, tf3*tf5))
+    assert tf / (tf + TF1) == Feedback(tf, TF1)
+    assert tf / (tf + TF1*TF2*TF3) == Feedback(tf, TF1*TF2*TF3)
+    assert TF1 / (tf + TF1*TF2*TF3) == Feedback(TF1, TF2*TF3)
+    assert (TF1*TF2) / (tf + TF1*TF2) == Feedback(TF1*TF2, tf)
+    assert (TF1*TF2) / (tf + TF1*TF2*tf5) == Feedback(TF1*TF2, tf5)
+    assert (TF1*TF2) / (tf + TF1*TF2*tf5*TF3) in (Feedback(TF1*TF2, tf5*TF3), Feedback(TF1*TF2, TF3*tf5))
     assert tf4 / (TransferFunction(1, 1, p) + tf4*tf6) == Feedback(tf4, tf6)
     assert tf5 / (tf + tf5) == Feedback(tf5, tf)
 
-    raises(ValueError, lambda: tf1*tf2*tf3 / (1 + tf1*tf2*tf3))
-    raises(ValueError, lambda: tf1*tf2*tf3 / tf3*tf5)
-    raises(ValueError, lambda: tf2*tf3 / (tf + tf2*tf3*tf4))
+    raises(ValueError, lambda: TF1*TF2*TF3 / (1 + TF1*TF2*TF3))
+    raises(ValueError, lambda: TF1*TF2*TF3 / TF3*tf5)
+    raises(ValueError, lambda: TF2*TF3 / (tf + TF2*TF3*tf4))
 
-    assert Feedback(tf, tf1*tf2*tf3).doit() == \
+    assert Feedback(tf, TF1*TF2*TF3).doit() == \
         TransferFunction((a2*s + p)*(s**2 + 2*s*wn*zeta + wn**2), k*(a2*p - s) + \
         (a2*s + p)*(s**2 + 2*s*wn*zeta + wn**2), s)
-    assert Feedback(tf1, tf2*tf3).doit() == \
+    assert Feedback(TF1, TF2*TF3).doit() == \
         TransferFunction((a2*s + p)*(s**2 + 2*s*wn*zeta + wn**2), (k*(a2*p - s) + \
         (a2*s + p)*(s**2 + 2*s*wn*zeta + wn**2))*(s**2 + 2*s*wn*zeta + wn**2), s)
-    assert Feedback(tf1*tf2, tf5).doit() == \
+    assert Feedback(TF1*TF2, tf5).doit() == \
         TransferFunction(k*(a0 + s)*(s**2 + 2*s*wn*zeta + wn**2), (k*(-a0 + a1*s**2 + a2*s) + \
         (a0 + s)*(s**2 + 2*s*wn*zeta + wn**2))*(s**2 + 2*s*wn*zeta + wn**2), s)
     assert Feedback(tf4, tf6).doit() == \
         TransferFunction(p*(p + s)*(a0*p + p**a1 - s), p*(p*(p + s) + (-p + s)*(a0*p + p**a1 - s)), p)
     assert -Feedback(tf4*tf6, TransferFunction(1, 1, p)).doit() == \
         TransferFunction(-p*(-p + s)*(p + s)*(a0*p + p**a1 - s), p*(p + s)*(p*(p + s) + (-p + s)*(a0*p + p**a1 - s)), p)
 
-    assert Feedback(tf1, tf2*tf5).rewrite(TransferFunction) == \
+    assert Feedback(TF1, TF2*tf5).rewrite(TransferFunction) == \
```

Comment:
For future reference, it would be best to move the capitalization of these variable names to a different PR, as it makes it difficult for the reader to see the changes that are actually related to the new TransferFunctionMatrix class.